buildsystem/python.cmake

```diff
@@ -169,7 +183,6 @@ function(add_cython_modules)
 				set(STANDALONE_NEXT FALSE)
 			else()
 				set_target_properties("${TARGETNAME}" PROPERTIES LINK_DEPENDS_NO_SHARED 1)
-				target_link_libraries("${TARGETNAME}" "${PYEXT_LINK_LIBRARY}")
```

Comment:
can't you add

```cmake
# Since this module is not embedded with python interpreter,
# Mac OS X requires a link flag to resolve undefined symbols
if(APPLE and NOT EMBED_NEXT)
	set_target_properties("${TARGETNAME}" PROPERTIES LINK_FLAGS "-undefined dynamic_lookup" )
endif()
```

below and leave this line? should have the same effect.